analysis/2026-05-15-regression/run_sweep.sh

```
@@ -0,0 +1,34 @@
+
+for i in 1 2 3 4 5; do
+  echo "==== $LABEL iter $i ===="
+  # Re-push logs (loki is in-memory but persists across runs of test framework
+  # because we skip its cleanup); be defensive anyway
+  poetry run python tests/llm/fixtures/test_ask_holmes/259_loki_historical_logs_pod_deleted_docker/generate_logs.py http://localhost:3259 > /dev/null 2>&1 || true
+
+  rm -f evals_report.md
+  timeout 300 poetry run pytest tests/llm/test_ask_holmes.py \
+    -k "259_loki_historical_logs_pod_deleted_docker" \
+    --no-cov -n0 -p no:cacheprovider --skip-setup --skip-cleanup \
+    > /tmp/sweep/${LABEL}_iter_${i}.log 2>&1 || true
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)